CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# dbt_hubspot v0.15.2
```

Comment:
Similar to the source PR, I believe we should consider making this a breaking change.

Reply:
If we do make this a breaking change we will also need to update the dbt_project.ymls and the README

Reply:
going with non breaking change for now, but updated the readme to include info about the config

Reply:
Makes sense to err on the side of caution. I went ahead and bumped it to v0.16.0 and highlighted the new tests under Breaking Changes. Also updated the readme version requirements and the project yml.